centos6-epel-rdt/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM centos:centos6
+
+RUN useradd docker
+
+RUN \
+  yum install -y epel-release && \
+  yum install -y xorg-x11-server-Xvfb R-devel
+
+RUN \
+  yum install -y centos-release-scl && \
+  yum install -y devtoolset-4-gcc-c++
+
+RUN echo "source /opt/rh/devtoolset-4/enable" >> /etc/bashrc
+
+ENV RHUB_PLATFORM linux-x86_64-centos6-epel-rdt
```

Comment:
Yes. The names is not the best, but it is for sysreqs, so it needs to be a platform from here: https://github.com/r-hub/sysreqsdb/tree/master/platforms

I think the `-rdt` image uses the same system requirements, so no new sysreqs platform is needed for this one.